src/types/people.ts

```diff
@@ -22,13 +22,13 @@ export interface PeopleInputInterface {
   address?: [AddressInputInterface];
 }
 export interface PeopleInterface {
-  id: string;
```

Comment:
That is wrong, all people must have, id, uuid, name, company, and user if Backend is not return it report the bug with Jessica

Reply:
ummm that gives a problem when I use the people interface to create a lead, I guess I should create a different interface for that or conditionalize this field.

Reply:
The ID is conditioned only when I need it